### PR TITLE
Added blog on Blockchain

### PR DESCRIPTION
**Pull request by kashish-51**
## Title :
Blockchain- Future of Tomorrow

## Issue no : 
#249

## Description
Added truncate

## Checklist
- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/tcet-opensource/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] I have read and followed the **Contributing Guidelines**

### CODE REVIEW

File: blog/2023-10-7-Blockchain/index.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+tags: [Blockchain, Data, Software, Technology ]
+description: 'Blockchain - Future of Tommorow'
+keywords: [Blockchain, Data, Software, Technology, Bitcoin ]
+---
+
+
+> Ever wondered what blockchain is and how it looks? Let's explore the realm of blockchain together. This blog focuses on the basics of blockchain technology along with its applications. In this evolving era, it is important to understand this technology because, according to  recent research, blockchain has the potential to reshape the future and the way we perceive it.
+
+<br />
+
```
Comment (vish-rt) [inline blog/2023-10-7-Blockchain/index.md:19]: Recommend [utilizing truncate](https://opensource.tcetmumbai.in/blog/blog-posting-guidelines#utilizing-truncate) to cut off the blog as it is overflowing the blogs page. Follow the posting guidelines on how to implement it.

**Review by vish-rt — CHANGES_REQUESTED** [review]

File: blog/2023-10-7-Blockchain/index.md (hunk 1 above)
Comment (kashish-51) [reply blog/2023-10-7-Blockchain/index.md:20]: Added truncate

**Comment by kashish-51** [comment]
I have added truncate. Please check now if everything is ok or not

**Comment by vish-rt** [comment]
Closing this Pull Request as it is created in the [wrong branch](https://github.com/tcet-opensource/documentation/issues/249#issuecomment-1748037909).
Kindly raise the PR to the `blogs` branch so the blog maintainers can review it.

**Pull request closed without merging**